packages/driver/src/cypress/error_messages.ts

```diff
@@ -865,6 +861,10 @@ export default {
       message: '`Cypress.Commands.add()` cannot create a new command named `{{name}}` because that name is reserved internally by Cypress.',
       docsUrl: 'https://on.cypress.io/custom-commands',
     },
+    invalid_new_query: {
+      message: '`Cypress.Commands.addQuery()` is used to create new query commands, but `{{name}}` is an existing Cypress command or is reserved internally by Cypress.\n\n If you want to override an existing command, use `Cypress.Commands.overrideQuery()` instead.',
+      docsUrl: 'https://on.cypress.io/custom-commands',
+    },
```

Comment:
addCommand will through the `reserved_command` error  if the command is reservered internally. Shouldn't we do the same for addQuery() so users understand how to handle this error?